src/main.rs

```diff
@@ -4,8 +4,11 @@ extern crate native;
 extern crate rsfml;
 extern crate collections;
 
-#[feature(phase)] extern crate log;
-#[phase(syntax, link)]
+// idk wth these lines were used for, couldn't find any other references
+// and the compiler complained. If the code doesn't use it maybe the
+// compiler does?
```

Comment:
I just want to add that these comments are just slightly off from my final conclusion (if you can call it that): that the program only benefited from their removal.